src/tools/cursors/MouseCursor.js

```diff
@@ -23,7 +23,7 @@ export default class MouseCursor {
   }
 
   /**
-   * getIconSVG - Returns an SVG of the icon only.
+   * GetIconSVG - Returns an SVG of the icon only.
```

Comment:
We can probably remove the name from the beginning of the description of each of these.

1. It's already included in the JSDoc meta
2. Forcing caps on comments for the first letter technically creates a casing issue with the method's name